cirq/ionq/job.py

```diff
@@ -154,15 +148,30 @@ def repetitions(self) -> Optional[int]:
         If run on the simulator this will return None.
 
         Raises:
-            IonQUnsuccessfulJob: If the job has failed, been canceled, or
-                deleted.
+            IonQUnsuccessfulJob: If the job has failed, been canceled, or deleted.
             IonQException: If unable to get the status of the job from the API.
         """
         self._check_if_unsuccessful()
         if 'metadata' in self._job and 'shots' in self._job['metadata']:
             return int(self._job['metadata']['shots'])
         return None
 
+    def measurement_dict(self) -> Dict[str, Sequence[int]]:
+        """Returns a dictionary of measurement keys to target qubit index."""
+        measurement_dict: Dict[str, Sequence[int]] = {}
+        if 'metadata' in self._job:
+            full_str = ''.join(
```

Comment:
Ah I see, I was suggesting a nested map, but I now see that we're restricting metadata to be key,value pairs of strings. 
This works then.

Reply:
I went back and forth.  It increases complexity, but increases size of circuits that the API works for.